fastly/utils.py

```diff
@@ -20,3 +20,23 @@ def purge_url(path):
         return response
 
     return None
+
+
+def purge_surrogate_key(key):
+    """
+    Purge a Fastly.com Surrogate-Key given a key.
+    """
+    if settings.DEBUG:
+        return
+
+    api_key = getattr(settings, 'FASTLY_API_KEY', None)
+    service_id = getattr(settings, 'FASTLY_SERVICE_ID', None)
```

Comment:
why getattr instead of grabbing directly as we do `DEBUG`?
`settings.FASTLY_API_KEY`

Reply:
copied from above function, but I think the point is that settings _may not_ have these values set at all, which would lead to an AttributeError. this approach is similar to a `{}.get(key, None)`

```
>>> from django.conf import settings
>>> settings.DOES_NOT_EXIST
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.12/site-packages/django/conf/__init__.py", line 104, in __getattr__
    val = getattr(_wrapped, name)
          ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Settings' object has no attribute 'DOES_NOT_EXIST'
>>> getattr(settings, 'DOES_NOT_EXIST', None)
>>> 
```